common/webui_url_constants.cc

```diff
@@ -7,7 +7,8 @@
 
 const char kAdblockHost[] = "adblock";
 const char kAdblockJS[] = "brave_adblock.js";
-const char kIPFSHost[] = "ipfs";
+const char kIPFSWebUIHost[] = "ipfs-internals";
+const char kIPFSWebUIURL[] = "chrome://ipfs-internals/";
```

Comment:
renamed kBraveUIIPFSURL -> kIPFSWebUIURL to be in one style

Reply:
better name 👍